src/components/common/OverallRating/index.js

```diff
@@ -5,7 +5,9 @@ import Thumbs from './Thumbs';
 import { Rating, RatingLabel } from './Rating';
 import styles from './Rating.module.css';
 
-const OverallRating = ({ rating, hasRatingLabel }) => {
+const OverallRating = ({ rating: rawRating, hasRatingLabel }) => {
+  const rating = Math.round(rawRating * 100) / 100;
```

Comment:
Suggestion: 可以用內建 toFixed
```suggestion
  const rating = Number(rawRating.toFixed(2));
```

Reply:
喔喔，對，toFixed 就可以四捨五入了，再轉 Number 就行，謝謝建議

Reply:
但為什麼是小數點第二位呀？萬一給小數點三位 Rating 會壞掉嗎？

Reply:
不會壞掉，在 calculateClipX 有再處理過

https://github.com/goodjoblife/GoodJobShare/blob/e1a48175178bc620a65a1435bbacc07f6f40b618/src/components/common/OverallRating/Thumbs.js#L32-L37

Reply:
如果不會壞掉，那是不是就不用小數點第二位了～

Reply:
我也好奇兩位數這是UI設計的嗎～
as a user 我覺得好像一位就好了..

Reply:
不好意思，重新確認一下 UI 稿，只有一位，我會再改成小數點後一位

<img width="899" alt="image" src="https://github.com/user-attachments/assets/f625523b-b379-491f-8890-ec24b2d5a7c3">

Reply:
我知道我困惑的點了，原來取小數點是為了 UI 顯示用

那我建議，在顯示 rating 文字的地方做處理就好，

在 `src/components/common/OverallRating/Rating.js`
```
const Rating = ({ rating, textYellow }) => (
  <div
    className={cn(styles.rating, {
      [styles.textYellow]: !textYellow,
    })}
  >
    {rating.toFixed(1))} 分
  </div>
);
```

原先 OverallRating 跟顯示幾位小數根本無關

Reply:
啊沒錯彥齊說的是對的，在最裡面做 toFixed 才合理，剛好也會轉成 string

Reply:
原來這樣做，不能保證印出來會是小數點兩位。已修正，改在最裡面做 toFixed 。